tools/envoy/publish.sh

```diff
@@ -27,8 +27,8 @@ PULP_DIST_NAME="alpine"
 [ -z "$PULP_PASSWORD" ] && msg_err "PULP_PASSWORD required"
 
 docker run --rm \
-        -e PULP_USERNAME=${PULP_USERNAME} \
-        -e PULP_PASSWORD=${PULP_PASSWORD} \
+        -e PULP_USERNAME="${PULP_USERNAME}" \
+        -e PULP_PASSWORD="${PULP_PASSWORD}" \
         -e PULP_HOST=${PULP_HOST} \
```

Comment:
Why no quotes for `PULP_HOST`?

Reply:
`shellcheck -f diff | patch -p1` took care of this file actually and `shellcheck` doesn't warn on this line. Perhaps since `PULP_HOST` is constant, it does some static analysis to see that it will expand without issue?

Odd, I'd want it to warn me anyway to prevent future bugs. Fixed.